influxdb_store.go

```diff
@@ -683,6 +747,25 @@ func addChildren(root *Trace, children []*Trace) error {
 	return nil
 }
 
+// pointMemoryUsage returns an estimate of the memory usage of the given *influxDBClient.Point
+// object in bytes.
+func pointMemoryUsage(v reflect.Value) int {
+	s := int(v.Type().Size())
+	switch v.Kind() {
+	case reflect.String:
+		s += v.Len()
+	case reflect.Ptr:
+		if !v.IsNil() {
+			s += pointMemoryUsage(reflect.Indirect(v))
+		}
+	case reflect.Map:
+		for _, key := range v.MapKeys() {
+			s += pointMemoryUsage(v.MapIndex(key))
+		}
+	}
```

Comment:
No, because the data type (https://godoc.org/github.com/influxdata/influxdb/client#Point) cannot contain one. (even the interface field is limited to these basic types)